addons/scene_palette/components/scene_thumbnail/scene_drop.gd

```diff
@@ -26,9 +26,21 @@ func show_file_label(show:bool):
 
 func _create_display_label(path:String) -> String:
 	var display_label = path.split('.tscn')[0].split('/')[-1]
+	if display_label == null:
+		display_label = path.split('.scn')[0].split('/')[-1]
```

Comment:
One way to fix it:
```suggestion
	var display_label = path.split('/')[-1]
	display_label = display_label.split('.tscn')[0]
	display_label = display_label.split('.scn')[0]
```

Reply:
Is there such a case where null can be returned? `split` splits by the specified delimiter, and just returns the whole string as an element of the array if there's nothing to split by, so if it doesn't have a `scn` or `tscn` extension, nothing happens.

Admittedly I should have just used `trim_suffix` for this instead, but didn't look it up at the time.